build.sbt

```diff
@@ -22,7 +25,23 @@ val javacSettings = Seq(
   "-Xlint:unchecked"
 )
 
-lazy val commonSettings = mimaDefaultSettings ++ Seq(
+lazy val mimaSettings = mimaDefaultSettings ++ Seq(
+  mimaBinaryIssueFilters ++= Seq(
+    // Added to have better parity between Java and Scala APIs
+    ProblemFilters.exclude[ReversedMissingMethodProblem]("play.libs.ws.StandaloneWSRequest.getBody"),
```

Comment:
@gmethvin this was a change to the interface (`StandaloneWSRequest`).

In this case, the problem is that Java API was missing some methods/features that are already present in the Scala API. Adding them breaks compatibility in both interface and implementation.

I don't see what would prevent custom implementations of the WS API.